packages/file/test/index.coffee

```diff
@@ -1138,10 +1138,45 @@ describe 'file', ->
           transform: ({config}) ->
             "#{config.content} world"
         .should.be.resolvedWith status: true
+
+    they 'transform content value', ({ssh}) ->
+      nikita
+        ssh: ssh
+        tmpdir: true
+      , ({metadata: {tmpdir}}) ->
+        @file
+          target: "#{tmpdir}/file"
+          content: 'hello'
+          transform: ({config}) ->
+            "#{config.content} world"
         @fs.assert
           target: "#{tmpdir}/file"
           content: 'hello world'
 
+    they 'transform returns null', ({ssh}) ->
+      nikita
+        ssh: ssh
+        tmpdir: true
+      , ({metadata: {tmpdir}}) ->
+        @file
+          target: "#{tmpdir}/file"
+          content: 'hello'
+          transform: ({config}) ->
+            null
+        .should.be.resolvedWith status: false
```

Comment:
what happens in such case, the file is not written ? good idea but it must be tested and documented